### PR TITLE
Connect SwaggerHub

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

File: content/en/docs/connect_manage_environ/_index.md [hunk 1]
```diff
@@ -91,6 +92,7 @@ For detailed information about agent configuration, features and limitations, se
 * [Discovery Agents for GitHub Repository](/docs/connect_manage_environ/connect_github_repository/)
 * [Discovery Agents for GitLab Repository](/docs/connect_manage_environ/connect_gitlab_repository/)
 * [Discovery Agents for Kafka Cluster](/docs/connect_manage_environ/connect_kafka_cluster/)
+* * [Discovery Agents for SwaggerHub](/docs/connect_manage_environ/connect_kafka_cluster/)
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/_index.md:95]: This link should point to a section about "Connect SwaggerHub" ?

**Review by robertengaxway — APPROVED** [review]
Assuming we can update the link to SwaggerHub doc.  I also added and had Shane review Embedded Azure as well.

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 8d4c228 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-440.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/connect_manage_environ/connect_swaggerhub/_index.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+* If the filters are not configured, all the files are discovered
+* The agent then creates an API service and revision to represent that specification file in Enterprise Marketplace
+* If the files are of a known specification type, then the service will be marked with that type. Otherwise, the service will have a type of `Unstructured`
+
+## Before you start
+
+* Read [Embedded SwaggerHub agent setup](/docs/connect_manage_environ/connect_swaggerhub_repository/embedded-agent-setup/)
+* Gather information on SwaggerHub:
+    * The api key that the agent will use to connect to SwaggerHub
+    * The oragization name that the agent will connect to
```
Comment (georgiana-caras) [inline content/en/docs/connect_manage_environ/connect_swaggerhub/_index.md:24]: typo

File: content/en/docs/integrate_with_central/cli_central/cli_embedded_agent_config.md [hunk 3]
```diff
@@ -150,3 +151,59 @@ Configured to set if headers should be processed, redact certain information, sa
 Redaction settings can be added to the Embedded Traceability Agent that will be used when finding and reporting transactional data to Enterprise Marketplace. The settings include the ability to customize the URL path of the transaction, the query arguments in the transaction, as well as the request and response headers.
 
 * Learn the Regular Expression syntax ([RE2 Syntax](https://github.com/google/re2/wiki/Syntax)) supported by the agent.
+
+### SwaggerHub
+
+* **owner** - the owner of the organization in SwaggerHub
+* **filter** - the filters to determine what kind of APIs to discover. Not setting the filter will discover all the APIs.
+  * **visibility** - the visbility state of the API in   SwaggerHub, it can be either public, private, or both.
```
Comment (georgiana-caras) [inline content/en/docs/integrate_with_central/cli_central/cli_embedded_agent_config.md]: API in   SwaggerHub: remove the extra space char

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Pull request merged**